pyp2rpm/convertor.py

```diff
@@ -56,7 +56,37 @@ def __init__(self, package=None, version=None,
         if not self.template.endswith('.spec'):
             self.template = '{0}.spec'.format(self.template)
         self.rpm_name = rpm_name
-        self.proxy = proxy
+        #check and set proxy settings
```

Comment:
Could this be a function/method on it's own?

Reply:
@hroncok I will look at doing that today. I was away on vaca sorry for slow response.